elixir/plug/app/lib/plug_example.ex

```diff
@@ -18,4 +18,11 @@ defmodule PlugExample do
   get "/error" do
     raise "Whoops!"
   end
+
+  get "/custom" do
```

Comment:
```suggestion
  get "/custom_instrumentation" do
```
Nit: Other apps use this route (`/custom` conflicted with Phoenix for some reason), so let's use the same route for all apps?